api-client-util/src/main/java/com/takipi/api/client/util/regression/RateRegression.java

```diff
@@ -54,15 +54,15 @@ public int compare(RegressionResult o1, RegressionResult o2) {
 
 	}
 
-	private static List<EventResult> getSortedNewEvents(Collection<EventResult> events) {
+	public static List<EventResult> getSortedNewEvents(Collection<EventResult> events) {
 
 		List<EventResult> result = Lists.newArrayList(events);
 
 		result.sort(new Comparator<EventResult>() {
 
 			@Override
 			public int compare(EventResult o1, EventResult o2) {
-				return (int) (o1.stats.hits - o2.stats.hits);
+				return (int) (o2.stats.hits - o1.stats.hits);
```

Comment:
Note this method is used in returning values all over this class, so changing the sort direction affects other places as well.

Is this change intended to change all of them as well?
If not, it should be reverted, and a new method (or overload with a boolean param for direction) should be used

Reply:
This is interesting. I found this code repeated 5 or so times in the API client. I am a fan of reusable code. The sort should bubble up the items with the most hits. I think is was a bug, but we should check